### DISCUSSION TITLE
Show bespoke message when all mechanics are busy and ride is broken

### DISCUSSION THREAD

**Discussion by mrtnptrs** [OpenRCT2/OpenRCT2#27060] [Feature requests]
**Situation when a mechanic is called for a broken ride:**

-When you did not hire any mechanics: you see at the repair tab of a ride "Calling mechanic...". This is strange, because there is no mechanic to call. Suggestion: in this situation let it display: "No mechanics hired yet." (So, here it should check IF any mechanics are hired. When not, it should display this)

-When you have many broken rides, but not enough mechanics (available); you see at the repair tab of a ride "Calling mechanic..." again. This should be something like: "No mechanic available to repair this ride at the moment." 
(So, here it should check IF any mechanics are hired, when yes, it should check if any mechanics have the status "walking" and display this message, as that is the only status of a mechanic when he is not (heading) for repairing a ride or doing an inspection.)

When mechanics are hired and at least one has the status "walking", "Calling mechanic..." should just be displayed.

What do you think?

**Comment by harrysbird** [comment]
Agreed .
I Like that idea to have this feature. 

I Thick it's Requires New Save Format for that.

**Comment by mrtnptrs** [comment]
@harrysbird Would it?: there would "only" be two checks needed to make a textual change. The "fix and mechanic calling status" should still be the same when saving your game. @IntelOrca @janisozaur Is he wrong or am I?

**Comment by janisozaur** [comment]
I don't see how it would require new save format, sounds doable within constraints of sv6.

**Comment by Gymnasiast** [comment]
First request has been implemented. Renamed the issue accordingly, and moving to Discussions.
